tests/misc/RwaATokenManager.t.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.0;
 
-import {IAccessControl} from 'src/contracts/dependencies/openzeppelin/contracts/IAccessControl.sol';
-import {IERC20} from 'src/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
-import {TestnetRwaERC20} from 'src/contracts/mocks/testnet-helpers/TestnetRwaERC20.sol';
-import {Errors} from 'src/contracts/protocol/libraries/helpers/Errors.sol';
-import {IPool} from 'src/contracts/interfaces/IPool.sol';
-import {IRwaAToken} from 'src/contracts/interfaces/IRwaAToken.sol';
+import {IAccessControl} from '../../src/contracts/dependencies/openzeppelin/contracts/IAccessControl.sol';
+import {IERC20} from '../../src/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+import {TestnetRwaERC20} from '../../src/contracts/mocks/testnet-helpers/TestnetRwaERC20.sol';
+import {Errors} from '../../src/contracts/protocol/libraries/helpers/Errors.sol';
+import {IPool} from '../../src/contracts/interfaces/IPool.sol';
+import {IRwaAToken} from '../../src/contracts/interfaces/IRwaAToken.sol';
 import {stdError} from 'forge-std/Test.sol';
 import {Vm} from 'forge-std/Vm.sol';
```

Comment:
do we need these 2 here btw?

Reply:
they are used in the file, yes: `stdError` is used in `test_fuzz_reverts_transferRwaAToken_NotEnoughBalance`, while `Vm` is used to extract recorded logs and assert no logs were emitted in `test_grantAuthorizedTransferRole_twice` and `test_fuzz_revokeAuthorizedTransferRole_NoEffect`